module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,15 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+std::vector<std::string> removeVowels(std::vector<std::string>& vec) {
+    std::vector<char> vowelVec{'a', 'e', 'i', 'o', 'u', 'y'};
+
+    for (std::string& str : vec) {
+        for (const char& vowel : vowelVec) {
```

Comment:
char is 1 byte size, reference to char is 4 or 8 byte size so in my opinion it's better to use pass by value
```suggestion
        for (const char vowel : vowelVec) {
```

Reply:
It doesn't mean that copying 1 byte instead of 4 or 8 is faster. Usually processors have instructions to copy whole register or word (4 or 8 bytes) so usually there is no difference. But it's a good point that we can safely copy it.